src/interface/mod.rs

```diff
@@ -1,7 +1,3 @@
-// Author: Nicholas Renner
```

Comment:
can we move all the removing commented out code in these files to a different pr?

Reply:
@Yaxuan-w